src/Kinect.cpp

```diff
@@ -22,9 +23,21 @@ float randomUniform(float min, float max)
 Kinect::Kinect()
 {
     // Set cam model
-    cam_model_.setFocalLengths(554.2559327880068, 554.2559327880068);
-    cam_model_.setOpticalCenter(320.5, 240.5);
-    cam_model_.setOpticalTranslation(0, 0);
+    sensor_msgs::CameraInfo cam_info;
+    cam_info.K = sensor_msgs::CameraInfo::_K_type({554.2559327880068, 0.0, 320.5,
+                          0.0, 554.2559327880068, 240.5,
+                          0.0, 0.0, 1.0});
+    cam_info.P = sensor_msgs::CameraInfo::_P_type({554.2559327880068, 0.0, 320.5, 0.0,
+                                                   0.0, 554.2559327880068, 240.5, 0.0,
+                                                   0.0, 0.0, 1.0, 0.0});
```

Comment:
Basically, you put the same numbers in a `PinholeCameraModel` and extract focal length, optical center and optical translation from there. 
* Why the change?
* Is the result any different?
* Is it still necessary to keep `cam_model_` as a member?

Reply:
The `depth_camera` is removed from Image, as it isn't used.

No the result isn't different.

Because I changed the same of the `depth_camera` from `cam_model_` to `rasterizer_`. But `cam_model_` it still used at the end of the file.

Reply:
Thanks, based on your answer I had a look at https://github.com/tue-robotics/rgbd/pull/26 and now I see your point. 

Seems a lot of code for this but then I guess that these parameters might be Kinect-specific.

Reply:
The matrices have a special structure which I filled in with the values which were already there.